handbook/product/design/research/user-testing-com.md

```diff
@@ -0,0 +1,82 @@
+# Usertesting.com
+
+## About
+
+Usertesting.com is _“a video-first platform where you can see and hear the experiences of real people as they engage with your products, designs, apps, processes, concepts, or brands.”_
+
+We utilize this platform to test prototypes, existing product features and perform surveys with users who are typically unfamiliar with Sourcegraph.
+
+## Test participants
+
+The service curates a pool of participants who are paid to conduct testing sessions. The participants chosen for your test must meet demographics and screening questions determined by the test administrator.
+
+Generally, our tests will use the pre-configured panel titled ‘General Audience’. Administers may also further refine this panel based on their unique requirements.
+
+> For instance, a test administrator testing concepts related to precise code intelligence may wish to exclude users who do not use typed languages to ensure results are relevant to the users of the feature.
+
+The platform also allows test administrators to provide participants, though we generally use [lookback](https://lookback.io/) for this purpose. This is because usertesting.com requires a software download many of our users do not wish to install.
```

Comment:
```suggestion
The platform also allows test administrators to provide participants, though we generally use [Lookback](https://lookback.io/) for this purpose, because usertesting.com requires a software download many of our users do not wish to install.
```